mopidy_jellyfin/ws_client.py

```diff
@@ -74,20 +75,22 @@ def run(self):
             on_error=lambda ws, error: self.on_error(ws, error))
         self.wsc.on_open = lambda ws: self.on_open(ws)
 
-
+        retry_count = 0
         while not self.stop:
 
-            self.wsc.run_forever(ping_interval=10)
-
-            if not self.stop:
-                break
+            time.sleep(retry_count * 5)
+            self.wsc.run_forever(ping_interval=10, ping_timeout=5)
 
-        self.callback('WebSocketDisconnect', None)
+            # If connection fails, attempt to reconnect every 60 seconds at max
+            max_tries = 12
+            if retry_count < max_tries:
+                retry_count += 1
 
+            if self.stop:
```

Comment:
What's the point of this? It's already the condition of the while loop

Reply:
It may not be needed.  I think I'm hitting a race condition here where sometimes it's been True, and others it's been False.  But I think you're right, it's not actually doing anything